templates/swifty-gtk4/template.json

```diff
@@ -16,12 +16,5 @@
       "default": "com.${USERNAME}.${NAMESPACE}",
       "pattern": "^\\w+(\\.\\w+)*$"
     }
-  },
```

Comment:
See other comment. This change needs to be reverted.

Reply:
Fixed